verilog/tools/ls/ls-wrapper.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-2022 The Verible Authors.
```

Comment:
Since the class is called VerilogLanguageServer, I'd call the file verilog-language-server.h (and the cc file and the library in the BUILD file).